base/applications/kbswitch/resource.h

```diff
@@ -7,5 +7,6 @@
 #define IDR_POPUP 12000
```

Comment:
Why is there such a large difference between the icons ID and the rest?

Reply:
Fixed as `100`. No problem.